src/forms/index.tsx

```diff
@@ -205,39 +228,73 @@ export function Select(props: RenderableProps<ISelectProps>) {
   });
   return (
     <div class={classes}>
-      <select id={props.id} name={props.id}>
+      <select
+        id={props.id}
+        name={props.id}
+        value={props.value}
+        onChange={props.onChange}
+        onFocus={props.onFocus}
+        onBlur={props.onBlur}
+        disabled={props.disabled}
+      >
         {props.options.map(el => (
-          <option>{el}</option>
+          <option key={el} value={el}>
+            {el}
+          </option>
         ))}
       </select>
     </div>
   );
 }
 
 export interface ICheckboxProps {
-  value?: boolean;
+  checked?: boolean;
   disabled?: boolean;
   onChanged?: (ev: Event) => void;
```

Comment:
@SolarLiner I think this should be renamed to `onChange`. It would break existing components though. Perhaps we leave this as optional, and add in `onChange: (ev: Event) => void;` until it's deprecated?

Reply:
+1. Must have been a typo, unless I've written that all over the code, in which case I'll blame English not being my native language :smile:

Reply:
But we should indeed add `onChange` and leave `onChanged`, marking it as deprecated.